src/modules/monotype.js

```diff
@@ -54,7 +55,9 @@ goog.scope(function() {
     var self = this;
     var projectId = self.configuration_['projectId'];
     var version = self.configuration_['version'];
-
+    //Expose 'mti_loadAllFonts' variable to load all fonts while using loader without webfontconfig
+    window['mti_loadAllFonts']=this.configuration_['loadAllFonts'];
```

Comment:
With webfontconfig setup, it is easy to get configuration without making any changes on webfontloader code. But with webfont.load method call, it is difficult to access this flag. As we're using this flag to decide whether to pass all fonts or only used fonts, we need to access this config.

We have also tried adding new function to pass configuration from loader to our publish code. But it was not working properly. So, we decided to exporting global variable.